web/src/js/components/FlowTable/FlowColumns.jsx

```diff
@@ -175,7 +176,9 @@ export function QuickActionsColumn({flow}) {
     return (
         <td className="col-quickactions">
             <div>
-                <div className="quickaction"><i className="fa fa-fw fa-ellipsis-h"/></div>
+                {selected ?
```

Comment:
is there a reason why we only want to show the quickactions menu for selected flows?

Reply:
The hover menu adds an interception filter which corresponds to the selected flow.
So, if we hover to a flow which is not selected, the hover menu still shows the interception filter suggestion of the selected flow.
This is very confusing.
So for now, I've made the menu appear only for the selected flow.